packages/completer/test/handler.spec.ts

```diff
@@ -320,5 +320,41 @@ describe('@jupyterlab/completer', () => {
         console.warn(editor.getCursorPosition());
       });
     });
+
+    it('should update cursor position after autocomplete on empty word', () => {
+      const model = new CompleterModel();
+      const patch = 'foobar';
+      const completer = new Completer({ editor: null, model });
+      const handler = new TestCompletionHandler({ completer, connector });
+      const editor = createEditorWidget().editor;
+      const text = 'eggs\n  # comment\nbaz';
+      const want = 'eggs\n foobar # comment\nbaz';
+      const line = 1;
+      const column = 1;
+      const request: Completer.ITextState = {
+        column: column,
+        line,
+        lineHeight: 0,
+        charWidth: 0,
+        coords: null,
+        text
+      };
+
+      handler.editor = editor;
+      handler.editor.model.value.text = text;
+      handler.editor.model.sharedModel.clearUndoHistory();
+      handler.editor.setCursorPosition({ line, column });
+      model.original = request;
+      const offset = handler.editor.getOffsetAt({ line, column });
+      model.cursor = { start: offset, end: offset };
+      // Make the completion, check its value and cursor position.
+      (completer.selected as any).emit(patch);
+      expect(editor.model.value.text).toBe(want);
+      expect(editor.getCursorPosition()).toEqual({
+        line,
+        column: column + 6
+      });
+      console.warn(editor.getCursorPosition());
```

Comment:
```suggestion
```

Reply:
Or not? Like we have this in a couple of places but it is a bit distracting to me so I would personally prefer to avoid adding new logs in tests.

Reply:
Agreed. Less is better. Since these logs are all over the place, I was thinking that some people use these for debugging. I simply copied the existing code so everything looks similar.